src/Build/Evaluation/PropertyTrackingEvaluatorDataWrapper.cs

```diff
@@ -318,16 +328,19 @@ private void TrackPropertyReassignment(P? predecessor, P property, string? locat
             }
 
             // Either we want to specifically track property reassignments
-            // or we do not want to track nothing - in which case the prop reassignment is enabled by default.
-            if ((_settings & PropertyTrackingSetting.PropertyReassignment) == PropertyTrackingSetting.PropertyReassignment ||
-                (_settings == 0 && ChangeWaves.AreFeaturesEnabled(ChangeWaves.Wave17_10)))
+            // or we do not want to track nothing - in which case the prop reassignment is enabled by default.    
+            if (PropertyTrackingUtils.IsPropertyTrackingEnabled(_settings, PropertyTrackingSetting.PropertyReassignment)
+                || (_settings == PropertyTrackingSetting.None && ChangeWaves.AreFeaturesEnabled(ChangeWaves.Wave17_14)))
```

Comment:
This is just refactoring with no functional change - correct?
Then the changewave value should have had stayed as it is - shouldn't it? Basically very soon we want the property reassignment tracking to be a default thing. Or do we want to push out that point in time more into the future?

Reply:
I changed a changewave due to your comment above https://github.com/dotnet/msbuild/pull/11106#discussion_r1904045137
:)